### ISSUE TITLE
Question about the paper.

### ISSUE THREAD

**Issue by BichenWuUCB** [hessamb/label-refinery#3]
Hi Hessam,

Thanks for sharing this great work! 

I have a question regarding the table2 of the paper. To generate the result in table 2, did you just use a regular pretrained resnet50 model as refinery? Or did you use a iteratively self-refined resnet50? If the former is true, then this is kind of similar to knowledge distillation?

Best,
Bichen

**Comment by hessamb** [comment]
Hi! Thank you for your interest. Yes, table 2 is generated based on a normal trained resnet-50, in a very similar manner to knowledge distillation. Please see #1 for label refinery's similarities and differences to knowledge distillation.
